Managers/MovableMan.h

```diff
@@ -586,6 +586,13 @@ class MovableMan : public Singleton<MovableMan>, public Serializable {
 
     bool IsOfActor(MOID checkMOID);
 
+    /// <summary>
+    /// Gives a unique, contiguous id per-actor. This is regenerated every frame. Returns -1 if the actor doesn't exist in our state.
```

Comment:
I think this or the member comment should mention that these ids are used for AI update throttling, otherwise it's kind of a weird thing to have, idk.

Also `Returns -1 if the actor doesn't exist in our state` should go into `<returns>`.